src/main/java/org/ngafid/events/EventDefinition.java

```diff
@@ -637,10 +637,17 @@ public static ArrayList<String> getAllNames(Connection connection, int fleetId)
      */
     public String toHumanReadable() {
         String text = "";
+
+        if (name.matches("^[AEIOU].*")) {
+            text = "An ";
+        } else {
+            text = "A ";
+        }
```

Comment:
Just a suggestion, you could do this in a ternary statement (i.e. `String text = (name.matches("^[AEIOU].*") ? "An " : "A ");`)

Reply:
Nice catch. Thanks